webdriver/tests/bidi/network/conftest.py

```diff
@@ -9,6 +10,7 @@
 RESPONSE_COMPLETED_EVENT = "network.responseCompleted"
 
 PAGE_EMPTY_HTML = "/webdriver/tests/bidi/network/support/empty.html"
+PAGE_EMPTY_TEXT = "/webdriver/tests/bidi/network/support/empty.txt"
```

Comment:
Mind defining and importing both from `network.__init__.py`?

Reply:
Done